src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends AbstractFigure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double calculatedArea() {
+        return 0.5 * (base1 + base2) * height;
+    }
```

Comment:
There is a typo in the method name 'calculatedArea'. It should be 'calculateArea' to follow standard naming conventions and to match the expected method name in the Figure interface.